### ISSUE TITLE
Bennon dialogue

### ISSUE THREAD

**Issue by fleenots** [nwn2fixes/player#32]
Hi
I found another bug i think. In Neverwinter nights the encounter with Leldon first thug Bennon. Even though I didn't join either the watch or Moira's gang I got two options in the dialogue. One to arrest Bennon (as I was part of the watch) and two telling that Benno's attack was not endorsed by Moira (as I was part of her gang). I guess conditional checks are in order in this dialogue.

**Comment by kevL** [comment]
good catch. i looked at it but unfortunately the fix isn't as easy as inserting conditions.

there is no "indeterminate" state of the gbWatch variable ... it's either on or off, with the ShadowThieves being the off state (by default). So even if the player hasn't made a decision to join the Watch or the ShadowThieves, he/she is assumed to be a member of the ShadowThieves in cases like this.

The dialog displays both nodes WITHOUT CONDITIONS at all.

I'm tempted to remove them both... leaving
- No, anyone who threatens you'll have to deal with me first.
- Whatever this is Neeshka, you handle it - it's not my concern.

but that would detract from the game, since player has /likely/ made a decision by that pt. A true fix, I believe, would involve either changing 'gbWatch' to a 3-state variable, or hunting down and making changes to any/every place that player can join the Thieves, and setting a different variable /positively/ to check for that also.

so i'm going to leave it as is for now, but will keep your ticket open,
